crates/bevy_ecs/src/world/unsafe_world_cell.rs

```diff
@@ -288,6 +288,17 @@ impl<'w> UnsafeWorldCell<'w> {
         &unsafe { self.world_metadata() }.components
     }
 
+    /// Retrieves this world's resource-entity map.
+    ///
+    /// # Safety
+    /// The caller must have exclusive read or write access to the resources that are updated in the cache.
+    #[inline]
+    pub unsafe fn resource_entities(self) -> &'w ResourceCache {
```

Comment:
These names should match. I think renaming `ResourceCache` to `ResourceEntities` makes sense.